services/subscribers/subscriber.go

```diff
@@ -104,14 +109,73 @@ func (i *s3Importer) ImportSubscribersFromFile(
 			if err != nil {
 				return fmt.Errorf("importer: marshal metadata: %w", err)
 			}
-			s.MetaJSON = metaJSON
+			sub.MetaJSON = metaJSON
 		}
 
-		err = storage.CreateSubscriber(ctx, s)
+		err = s.db.CreateSubscriber(sub)
 		if err != nil {
 			return fmt.Errorf("importer: create subscriber: %w", err)
 		}
 	}
 
 	return
 }
+
+func (s *service) RemoveSubscribersFromFile(
+	ctx context.Context,
+	filename string,
+	userID int64,
+) (err error) {
+	res, err := s.client.GetObject(&s3.GetObjectInput{
```

Comment:
Can we add the ReadCloser as an argument in this method here, we can make it similar as the ImportSubscribersFromFile. The benefits from this are that we can test these methods more easily, since we won't have the s3 dependency.